components/HomepageSubnav.tsx

```diff
@@ -0,0 +1,48 @@
+import Link from 'next/link';
+import styles from '/styles/HomepageSubnav.module.scss';
+
+type HomepageSubnavProps = {
+    title: string;
+    jobs: number;
+    children: React.ReactNode;
+};
+
+export const HomepageSubnav = ({
+    title,
+    jobs,
+    children,
+}: HomepageSubnavProps) => (
+    <div className={styles.container}>
+        <ul className={styles.navigation}>
+            <li className={styles.navItem}>
+                <a className={styles.navLink} href="#benefits">
+                    Benefits
+                </a>
+            </li>
+            <li className={styles.navItem}>
+                <a className={styles.navLink} href="#lifeAsATorchboxer">
+                    Life as a Torchboxer
+                </a>
+            </li>
+            <li className={styles.jobNavItem}>
+                <Link href="/jobs">
+                    <a className={styles.jobsLink}>
+                        Jobs
+                        {jobs !== undefined && jobs > 0 ? (
+                            <div className={styles.jobsBadge}>{jobs}</div>
+                        ) : null}
+                    </a>
+                </Link>
+            </li>
+        </ul>
+        <img
+            className={styles.image}
+            src="images/arrow-loop-right.svg"
```

Comment:
**Chore:** remove content shifting


**Suggestion:**
I've tried to demonstraight how this can cause content shifting in the screenshots below due to having to fetch the svg file.

There are a few ways to prevent this, one would be by defining height and width addtributes on this element (i.e. 123 and 95) which we'd have to do if this was anything but an SVG file. However since it is an SVG we can either create a react component and import it here (SVG's can be rendered on the page so no need to load it in an image element.), or we can  use something like [SVGR](https://react-svgr.com/docs/next/) to do that for us.

```js
import React from 'react';

const ArrowLoopImage = ({
    className?: string,
    width: int,
    height: int,
    //or we can pass in height and width in props below
    props: React.SVGProps<SVGSVGElement>,
}): React.ReactElement => (
    <svg
        xmlns="http://www.w3.org/2000/svg"
        width={width}
        height={height}
        viewBox="0 0 95 123" // make sure this is included in your svg file you're using
        fill="none" 
        /* eslint-disable-next-line react/jsx-props-no-spreading */
        {...props}
    >
        <path
            fill="#E0107F"
            d="M8 14.5l6-4.5-6-4.5v9zM10 0C4.48 0 0 4.48 0 10s4.48 10 10 10 10-4.48 10-10S15.52 0 10 0zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
        />
    </svg>
);

export default ArrowLoopImage;
```

<img width="399" alt="Screenshot 2022-01-26 at 14 04 22" src="https://user-images.githubusercontent.com/35934884/151190525-98cea9d6-a3e7-444d-9510-97a8669be167.png">
<img width="390" alt="Screenshot 2022-01-26 at 14 04 27" src="https://user-images.githubusercontent.com/35934884/151190543-726ce9cf-1769-4538-a333-6b1d13f317c5.png">

Reply:
SVGR looks cool but it works suspiciously like magic; I'd rather have the control of making a custom component here. Good point about the image loading in. I'm using [svg2jsx.com](svg2jsx.com) to convert the image, it gives a decent starting point.

Reply:
I've gone for adding width, height and classname as the SVG props - if this arrow gets reused we can see about expanding these props there, to the current designs this arrow isn't reused.